app/src/main/java/io/apicurio/registry/ccompat/rest/impl/CompatibilityResourceImpl.java

```diff
@@ -68,9 +68,8 @@ public CompatibilityCheckResponse testCompatibilityBySubjectName(
             SchemaContent request) throws Exception {
 
         CompatibilityCheckResponse result = facade.parseVersionString(subject, versionString, version -> {
-            // TODO are we safe to assume AVRO?
             try {
-                rules.applyRule(subject, version, ArtifactType.AVRO, ContentHandle.create(request.getSchema()));
+                rules.applyRule(subject, version, ArtifactType.fromValue(request.getSchemaType()), ContentHandle.create(request.getSchema()));

```

Comment:
Are Confluent's schema type constants the same as ours??

Reply:
Yep, same constant values.